cypress-custom/integration/fee.test.ts

```diff
@@ -45,6 +53,9 @@ describe('Fee endpoint', () => {
 
 describe('Fetch and persist fee', () => {
   beforeEach(() => {
+    // set the Cypress clock to now
+    // only override Date functions
+    cy.clock(new Date().getTime(), ['Date'])
```

Comment:
I thought the goal to mock the time was to set to a fixed time, not `now`.
Such as `new Date(2020,01,01)` or something.
Then you can compare against also a const date `new Date(2020, 01, 01, 04)`. Or just add 4h to the const date.

Anyway, I assume this works, just saying :)

BTW, `new Date().getTime() === Date.now()`